modules/social_features/social_branding/src/Wrappers/Color.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Drupal\social_branding\Wrappers;
+
+use Spatie\Color\Hex;
+
+/**
+ * A representation of theming color.
+ */
+class Color {
+
+  /**
+   * The color as hexadecimal string..
+   *
+   * @var string
+   */
+  private string $hex;
+
+  /**
+   * The color representation as individual RGBA components.
+   *
+   * @var \Drupal\social_branding\Wrappers\RGBAColor
+   */
+  private RGBAColor $rgba;
+
+  /**
+   * The color representation that's valid in css style rules.
+   *
+   * @var string
+   */
+  private string $css;
+
+  /**
+   * Create a new Color instance.
+   *
+   * @param string $color
+   *   The determined color.
+   */
+  public function __construct(string $color) {
+    $hexColor = Hex::fromString($color);
+
+    $this->hex = $hexColor->red() . $hexColor->green() . $hexColor->blue();
+    $this->rgba = new RGBAColor($color);
+    $this->css = $hexColor->__toString();
+  }
+
+  /**
+   * Get the Hex color.
+   *
+   * @return string
+   *   The color as hexadecimal string.
+   */
+  public function getHex() : string {
```

Comment:
Lets change this to `hexRGB` because we know from our schema that we may add a `hexRGBA` color/method in the future :) We also know that this method never provides the a Hex string containing alpha.